packages/polaris-viz/CHANGELOG.md

```diff
@@ -5,7 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-<!-- ## Unreleased -->
+## Unreleased
+
+### Added
+
+- Added `useTrendIndicators` prop to `SimpleNormalizedBarChart` to support rendering the `TrendIndicator` component. This allows for a more flexible representation of data trends, accommodating designs that require more detailed trend indicators.
```

Comment:
I'm not sure if this is the right approach, but we cannot replace the chevrons with the trend indicator as it would disrupt the existing reports' experience. Also, I don't think we can replace the breakdown bar viz to use a stacked horizontal bar instead of a simple normalized bar 🤷‍♀️ 
I'm open to other suggestions.

cc @envex @tylernoseworthy